ex3/ex3.c

```diff
@@ -10,6 +10,13 @@
 int main(void)
 {
     // Your code here
-
+    int caller = fork();
+    if(caller < 0){
+        printf("fork failed");
+    }else if(caller == 0){
+        printf("This is the Parent: Goodbye\n");
+    }else{
+        printf("This is the Child: Hello\n"); 
```

Comment:
This is the parent. you need to add a `wait(NULL)` or a `waitpid(caller, NULL, 0);` to make the parent wait.